### PR TITLE
Progress report - Arpil 2023

### PR DESCRIPTION
**Pull request by goldenx86**
I said it was gonna be short...

### CODE REVIEW

File: site/content/entry/yuzu-progress-report-apr-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+- Improving consistency for `Query Cache` asynchronous downloads
+
+The results are amazing. Most games that used to need high GPU accuracy to look right can now run on normal with no issues.
+Plus, all this wizardry reduces bandwidth usage, boosting performance up to 50% for everyone, from the low-end APUs to the high-end beasts.
+
+Here’s an incomplete list of changes:
+
+- As noted previously, many games which required high GPU accuracy to be visually accurate now work with normal GPU accuracy with minimal sacrifice.
+- Particles in `The Legend of Zelda: Breath of the Wild` have been fixed, resulting in 40% higher performance with accurate rendering at normal GPU accuracy.
+- Tree flickering in `The Legend of Zelda: Breath of the Wild` has been fixed on all GPU accuracy options.
```
Comment (liamwhite) [inline site/content/entry/yuzu-progress-report-apr-2023/index.md:35]: Recommend moving these two later in the list

File: site/content/entry/yuzu-progress-report-apr-2023/index.md [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+Still on fire, and with more work to come, Blinkhawk hasn’t stopped.
+For something light, he decided to refactor a big part of [Accelerate DMA](https://yuzu-emu.org/entry/yuzu-progress-report-feb-2023/#project-yfc-175) to do {{< gh-hovercard "10082" "texture downloads" >}} through the texture cache instead.
+The result is a return to the performance `Pokémon Sword & Shield` and `Hyrule Warriors: Age of Calamity` had before the old Y.F.C 1.75 changes.
+Add up the gains from Y.F.C 1.90, and you have a winner for low end systems!
+
+We have many more changes Blinkhawk implemented, but they didn’t make it in time for the deadline for the progress report, so we will be mentioning them next month.
+
+## General changes and settings glossary
+
+The modding community requested us to allow for larger texture mods. The old 6GB limit was not enough, so byte[] {{< gh-hovercard "10035" "expanded it to support the 8GB arrangement" >}} instead, giving almost all of it to the emulated program.
```
Comment (liamwhite) [inline site/content/entry/yuzu-progress-report-apr-2023/index.md:159]:
```suggestion
The modding community requested us to allow for larger texture mods. The old 6GB limit was not enough, so byte[] {{< gh-hovercard "10035" "expanded it to support a custom 8GB arrangement" >}} instead, giving almost all of it to the emulated program.
```

File: site/content/entry/yuzu-progress-report-apr-2023/index.md [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+	"./igs.png| Nothing beats the classics (IGS Classic Arcade Collection)"
+  >}}
+
+One of the options available to the users with [LDN](https://yuzu-emu.org/entry/ldn-is-here/) is the ability to [create private rooms](https://yuzu-emu.org/help/feature/multiplayer/), providing hosts with several options to set their servers as they want.
+
+[twitchax](https://github.com/twitchax) knows that for certain scenarios like `fly.io`, a {{< gh-hovercard "10068" "custom bind address" >}} is needed.
+They implemented the functionality and now users can pass the `--bind-address` argument to the room’s configuration.
+Thanks!
+
+Continuing his work on improving CPU timing accuracy, [Morph](https://github.com/Morph1984) fixed confusion between the raw CPU frequency and `Counter-timer Frequency`, or CNTFRQ.
```
Comment (liamwhite) [inline site/content/entry/yuzu-progress-report-apr-2023/index.md:200]: Missing a link to the PR

File: site/content/entry/yuzu-progress-report-apr-2023/index.md [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+We reported the crash to Intel six months ago, and their reply this month was:
+
+> Thank you for reporting this issue. Our priority is to target the most popular games and apps to focus our efforts on providing a high quality, stable experience for the broadest set of users. We will continue to improve our software performance and compatibility throughout 2023 and beyond.
+
+So even after waiting six months with a test case provided, the reason explained, and even after pointing out where in the driver the crash happens, Intel can’t find the time to fix the issue..
+This is some kind of twisted joke.
+For comparison, when you do this with NVIDIA, they *hire* you.
+
+We’re left with no option but to mitigate the crashes as much as we can. 
+To do so, Morph {{< gh-hovercard "10110" "disabled compute shaders" >}} entirely for Intel Windows drivers, ensuring games like `The Legend of Zelda: Breath of the Wild` can even be booted, let alone playable.
```
Comment (liamwhite) [inline site/content/entry/yuzu-progress-report-apr-2023/index.md:343]:
```suggestion
To do so, Morph {{< gh-hovercard "10110" "disabled compute shaders" >}} entirely for Intel Windows drivers, ensuring games like `The Legend of Zelda: Breath of the Wild` can still boot―even though they'll have gameplay issues as a result.
```

File: site/content/entry/yuzu-progress-report-apr-2023/index.md [hunk 5]
```diff
@@ -0,0 +1,381 @@
...
+- Vertex explosions in `Persona 5 Royal` no longer occur with Normal GPU accuracy, resulting in an up to 30% increase in performance.
+- `Atelier Ryza` series games now render correctly.
+- The pessimistic flushes option in advanced graphics settings is no longer needed in any of the affected games it benefitted and we have now removed it.
+- `Mortal Kombat 11` no longer has any vertex explosions.
+- `NieR:Automata The End of YoRHa Edition` now renders correctly.
+- `Bayonetta 3` no longer requires High GPU accuracy to render correctly.
+- `Splatoon 2`’s ink physics work correctly on AMD GPUs while using High GPU accuracy.
+- Particles in `The Legend of Zelda: Breath of the Wild` have been fixed, resulting in 40% higher performance and accurate rendering on Normal GPU accuracy.
+- Tree flickering in `The Legend of Zelda: Breath of the Wild` has been fixed on all GPU accuracy options.
+- And much, much more!.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-apr-2023/index.md:52]:
```suggestion
- And much, much more!
```

File: site/content/entry/yuzu-progress-report-apr-2023/index.md [hunk 6]
```diff
@@ -0,0 +1,381 @@
...
+Apparently the game doesn’t do a very good job implementing its audio code on the Switch, and fails to initialize a new audio output session, which should return an error.
+Properly {{< gh-hovercard "10056" "returning this error as a result" >}} is enough to get the game working.
+
+{{< imgs
+	"./igs.png| Nothing beats the classics (IGS Classic Arcade Collection)"
+  >}}
+
+One of the options made available to our [LDN](https://yuzu-emu.org/entry/ldn-is-here/) users is the ability to [create private rooms](https://yuzu-emu.org/help/feature/multiplayer/), providing hosts with several options to configure their servers as they want.
+
+[twitchax](https://github.com/twitchax) knows that for certain hosting servers, such as `fly.io`, a {{< gh-hovercard "10068" "custom bind address" >}} is needed.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-apr-2023/index.md:200]:
```suggestion
[twitchax](https://github.com/twitchax) knows that for certain server hosts, such as `fly.io`, a {{< gh-hovercard "10068" "custom bind address" >}} is needed.
```

File: site/content/entry/yuzu-progress-report-apr-2023/index.md [hunk 7]
```diff
@@ -0,0 +1,381 @@
...
+
+{{< imgs
+	"./dxgi.png| Don’t forget to set up your NVIDIA control panel accordingly!"
+  >}}
+
+While this fun renaming works fine, we also heard from Windows Insider users that future versions of the Xbox Game Bar application will allow yuzu to work with Auto HDR without doing any renaming. 
+This will most likely require newer NVIDIA drivers to work.
+
+If you own an HDR-compatible monitor and want to compare the results, here’s a .jxr [HDR capture](https://github.com/goldenx86/yuzu-emu.github.io/blob/apr23/site/content/entry/yuzu-progress-report-apr-2023/hdr.jxr) that the default Windows 11 photo viewer can open, and a comparable .png native [SDR capture](https://github.com/goldenx86/yuzu-emu.github.io/blob/apr23/site/content/entry/yuzu-progress-report-apr-2023/sdr.png).
+You will of course need HDR output enabled in both your display’s and Windows 11’s settings.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-apr-2023/index.md:290]:
```suggestion
You will of course need HDR output enabled in both your display’s and Windows 11 settings.
```

**Review by Schplee — CHANGES_REQUESTED** [review]

**Pull request merged**